install_sct

```diff
@@ -621,7 +621,9 @@ if [ "$ACTUAL_PYTHON" != "$EXPECTED_PYTHON"  ]; then
 fi
 
 # Ensure that packaged GLIBC version is up to date (https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3927)
-conda install -y -c conda-forge libstdcxx-ng
+if [[ $OS == linux* ]]; then
```

Comment:
Is there a reason to use the pattern `linux*` rather than just the literal `linux`, here and on [line 577](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/7eaadeed1b415889fe03cfb846459a5450d18f7f/install_sct#L577)? As far as I can tell, the only possible values for `$OS` are the literals [`osx`](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/7eaadeed1b415889fe03cfb846459a5450d18f7f/install_sct#L167) and [`linux`](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/7eaadeed1b415889fe03cfb846459a5450d18f7f/install_sct#L172).

Reply:
Ahh, thank you for asking about this! 

I had noticed than an earlier line in the script mentions `linux*`:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/6c3a3dc2bcc819dddf03026e45fe4b61c1f1282b/install_sct#L576-L583

And I had presumed that there were multiple `linux`-es? But, as you've pointed out, there aren't! 

_(At least, not anymore! Doing a `git blame` on that original `linux*` line shows that it in a much older iteration of `install_sct`, we used to treat `centos6` as a separate case from other `linux` distros.)_

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/95954e209444eea033c066404149fbf726777c33/install_sct#L248-L257

I can change both instances of `linux*` to `linux`. :)